homeassistant/components/hausbus/gateway.py

```diff
@@ -0,0 +1,185 @@
+"""Representation of a Haus-Bus gateway."""
+
+import asyncio
+from collections.abc import Callable, Coroutine
+from typing import Any, cast
+
+from pyhausbus.ABusFeature import ABusFeature
+from pyhausbus.BusDataMessage import BusDataMessage
+from pyhausbus.de.hausbus.homeassistant.proxy.Controller import Controller
+from pyhausbus.de.hausbus.homeassistant.proxy.controller.data.Configuration import (
+    Configuration,
+)
+from pyhausbus.de.hausbus.homeassistant.proxy.controller.data.ModuleId import ModuleId
+from pyhausbus.de.hausbus.homeassistant.proxy.controller.data.RemoteObjects import (
+    RemoteObjects,
+)
```

Comment:
I've never seen this Java-style package naming in a Python library 🤷‍♂️

Reply:
The creator of the library is a Java-developer. I think he generated most of the python library from his existing code base.